src/common/components/auth/ChangePassword.tsx

```diff
@@ -95,11 +92,11 @@ function ChangePassword() {
 	];
 
 	const onSubmit = async (values: IPasswordReset) => {
-		const { oldPassword, newPassword } = values;
-		await mutate({
-			oldPassword,
-			newPassword,
-		});
+		// const { oldPassword, newPassword } = values;
```

Comment:
Let's remove this comment if code is no longer necessary please

Reply:
fixed